docs/features/linux_networking/es2k/readme-linux-networking-ipv6.md

```diff
@@ -0,0 +1,264 @@
+<!--
+/*
+ * Copyright (c) 2023 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+- -->
+
+## Table of contents
+
+1. [Overview](#overview)
+2. [Topology](#topology)
+3. [Create P4 artifacts and start Infrap4d process](#create_p4_artifacts_and_start_infrap4d_process)
+4. [Steps to create the topology](#steps)
+5. [Limitations](#limitations)
+
+## Overview <a name="overview"></a>
+
+This README describes a step-by-step procedure to run the Linux networking scenario on ES2K.
+
+## Topology <a name="topology"></a>
+
+<p align="center">
+  <img alt="Linux Networking Topology" src="topology-linux-networking.PNG">
+</p>
+
+
+* Notes about topology:
+  * Four Kernel netdevs are created by default by loading IDPF driver during ACC bring-up. You can also create more than Four netdevs. For that, we need to modify `acc_apf` parameter under `num_default_vport` in `/etc/dpcp/cfg/cp_init.cfg` on IMC before starting `run_default_init_app`.
+  * In `/etc/dpcp/cfg/cp_init.cfg` file also modify default `sem_num_pages` value to the value mentioned in `/opt/p4/p4sde/share/mev_reference_p4_files/linux_networking/README_P4_CP_NWS`.
+  * vlan1, vlan2, .... vlanN created using Linux commands and are on top of an IDPF Netdev. These VLAN ports should be equal to number of VM's that are spawned.
+  * br-int, VxLAN ports are created using ovs-vsctl command provided by the networking recipe and all the vlan ports are attached to br-int using ovs-vsctl command.

```

Comment:
Can we separate 2nd part of this sentence?

All VLAN port representors and VxLAN ports are part of same broadcast domain (br-int)?